cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/LedgerEvents/Gov/PredefinedAbstainDRep.hs

```diff
@@ -133,8 +140,25 @@ delegateToAlwaysAbstain
   -> PaymentKeyInfo
   -> StakingKeyPair
   -> m ()
-delegateToAlwaysAbstain execConfig epochStateView configurationFile socketPath sbe work prefix
-                        payingWallet skeyPair@(StakingKeyPair vKeyFile _sKeyFile) = do
+delegateToAlwaysAbstain execConfig epochStateView configurationFile socketPath sbe work prefix =
+  delegateToAutomaticDRep execConfig epochStateView configurationFile socketPath sbe work prefix
+                          "--always-abstain"
+
+delegateToAutomaticDRep
+  :: (MonadTest m, MonadIO m, H.MonadAssertion m, MonadCatch m)
```

Comment:
```suggestion
  :: (HasCallStack, MonadTest m, MonadIO m, H.MonadAssertion m, MonadCatch m)
```

Reply:
Fixed in: https://github.com/IntersectMBO/cardano-node/pull/5809/commits/5e708b0aca65f6c9b534fa62d65efa163b8f184f